hack/docs-requirements.txt

```diff
@@ -0,0 +1 @@
+mkdocs==1.3.1
```

Comment:
I think they are up to 1.4.z now. Do you think we can use: `mkdocs==1.4` so we always get latest minor?

Reply:
@bostrt it will require `python >= 3.8` to support `mkdocs>=1.4`. Changing it, may we need to mention it somewhere?
Maybe on the `dev.md` ?

Reply:
Done https://github.com/redhat-openshift-ecosystem/provider-certification-tool/pull/24/commits/af6e77240cec17e2e4b21713dc2fdd5082b29fdf